src/OctoshiftCLI.Tests/GithubApiTests.cs

```diff
@@ -1832,45 +2030,41 @@ public async Task GetUserId_Returns_The_User_Id()
         }
 
         [Fact]
-        public async Task GetUserId_For_No_Existant_User_Returns_Null()
```

Comment:
We now just cover the generic "there is an error" case rather than the "user not found" case - although the test proves how we handle a missing user in particular